crt_portal/cts_forms/tests/test_views.py

```diff
@@ -876,6 +876,92 @@ def test_complaint_id_filter(self):
         self.assertTrue('2 records' in str(response.content))
 
 
+class CRT_Disposition_Tests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.superuser = User.objects.create_superuser('superduperuser', 'a@a.com', '')
+        self.report_data = SAMPLE_REPORT_1.copy()
+        self.report_data.update({'status': 'closed'})
+        closed_date = "2022-02-01 18:17:52.74131+00"
+        self.report_data.update({'closed_date': closed_date})
+        retention_schedule = RetentionSchedule.objects.get(retention_years=10)
+        self.report_data.update({'retention_schedule': retention_schedule})
+        expiration_date = "2032-02-01 18:17:52.74131+00"
+        self.report_data.update({'expiration_date': expiration_date})
+        self.report_data.update({'eligible_date': "2032-01-01 18:17:52.74131+00"})
+        self.report = Report.objects.create(**self.report_data)
+        self.report_data_2 = SAMPLE_REPORT_1.copy()
+        self.report_data_2.update({'status': 'closed'})
+        self.report_data_2.update({'closed_date': closed_date})
+        retention_schedule_2 = RetentionSchedule.objects.get(retention_years=3)
+        self.report_data_2.update({'retention_schedule': retention_schedule_2})
+        expiration_date_2 = "2025-02-01 18:17:52.74131+00"
+        self.report_data_2.update({'expiration_date': expiration_date_2})
+        self.report_data_2.update({'eligible_date': "2025-01-01 18:17:52.74131+00"})
+        self.report_2 = Report.objects.create(**self.report_data_2)
+        self.report_data_3 = SAMPLE_REPORT_1.copy()
+        self.report_data_3.update({'status': 'closed'})
+        self.report_data_3.update({'closed_date': closed_date})
+        retention_schedule_3 = RetentionSchedule.objects.get(retention_years=1)
+        self.report_data_3.update({'retention_schedule': retention_schedule_3})
+        expiration_date_3 = "2023-02-01 18:17:52.74131+00"
+        self.report_data_3.update({'expiration_date': expiration_date_3})
+        self.report_data_3.update({'eligible_date': "2023-01-01 18:17:52.74131+00"})
```

Comment:
Do you want to be updating expiration_date and eligible_date manually? I would expect the tests to be relying on the calculations in `save()` (or in a query expression, if you switch to that)